src/storage/cubrocks.cpp

```diff
@@ -115,15 +137,52 @@ cubrocks::context::kv_tran_start (int tran_index)
   rocksdb::WriteOptions write_options;
 
   assert (tran_index < MAX_NTRANS);
-  assert (sessions[tran_index].active);
+  assert (transactions[tran_index].active);
+  assert (transactions[tran_index].txn == nullptr);
+
+  transactions[tran_index].txn = db->BeginTransaction (write_options);
```

Comment:
rocksdb에서 발생한 문제들이 CUBRID로 전이되지 못하도록 우선 assert()로 다 막는게 문제 발생 시 찾기 편한 것 같습니다.
```suggestion
  transactions[tran_index].txn = db->BeginTransaction (write_options);
  assert (transactions[tran_index].txn != nullptr);
```